instance/static/html/instance/index.html

```diff
@@ -24,6 +24,11 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="right" tooltip="Provisioning in progress" ng-switch-default></i>
+          <i class="fa fa-times" tooltip-placement="right" tooltip="Provisioning failure" ng-switch-when="provisioning_failure"></i>
+          <i class="fa fa-check" tooltip-placement="right" tooltip="Ready" ng-switch-when="ready"></i>
+        </div>
```

Comment:
Cf my comment for https://github.com/open-craft/opencraft/pull/13/files#diff-772e1d190b3c56d74ec801abe130ba8cR50 - maybe the approach from https://github.com/open-craft/opencraft/pull/13/files#diff-848aab5153020ca3817d6db9d4d3a144R91 would help with this too.